### ISSUE TITLE
Update Disable / Enable functions

### ISSUE THREAD

**Issue by Gaffen** [guidobouman/jquery-panelsnap#148]
I have a use case where I'd like to disable and re-enable PanelSnap. As it stands the (undocumented) enable/disable functions work for this, however the library will continue to log to console as the events are still attached to the scroll container.

Would it be possible to add removing/re-adding the relevant events to the disable and enable methods?

**Comment by guidobouman** [comment]
That's a good idea. Aside from the logging to the console, does the plugin do anything else you don't want it to? I'm wondering how this would work and what the actual use would be.

You are of course welcome to submit a PR. Otherwise I'll look into this when I have time.

**Comment by Gaffen** [comment]
@guidobouman No, the plugin is fully functional for my use case as it is; so this is purely an enhancement.

The use case in question is that I have elements below the PanelSnap instance, which I want to be able to scroll to. I'm using an intersection observer to toggle the enabled/disabled status of PanelSnap as a result.

I will look into submitting a pull request when I get a moment!
